stage1_fly/enter/main.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func main() {
+	log.Printf("Not doing anything here! (%+v)", os.Args)
```

Comment:
Instead of adding a new file `stage1_fly/enter/main.go` that does nothing, I suggest to patch `stage0/enter.go` to return "not implemented":

```
        ep, err := getStage1Entrypoint(cdir, enterEntrypoint)
        if err != nil {
                return fmt.Errorf("error determining 'enter' entrypoint: %v", err)
        }
        if ep == "" {
                return fmt.Errorf("enter not implemented in this stage1")
        }
```